node/deny.toml

```diff
@@ -59,7 +59,7 @@ skip = [
     { name = "regex-syntax", version = "=0.6.29" },
 
     # Old versions required by hyper.
-    { name = "socket2", version = "=0.4.9" },
+    { name = "socket2", version = "=0.4.10" },
```

Comment:
Nit: You can use a caret requirement here (`version = "^0.4"`) to make it more future-proof.

Reply:
I don't think we want it to be future proof in that sense.

Reply:
Fixed.